patterns/roks/variables.tf

```diff
@@ -163,6 +163,42 @@ variable "entitlement" {
   default     = null
 }
 
+variable "disable_public_endpoint" {
+  type        = bool
+  description = "Flag indicating that the public endpoint should be disabled"
+  default     = true
+}
+
+variable "verify_worker_network_readiness" {
+  type        = bool
+  description = "By setting this to true, a script will run kubectl commands to verify that all worker nodes can communicate successfully with the master. If the runtime does not have access to the kube cluster to run kubectl commands, this should be set to false."
+  default     = false
+}
+
+variable "logdna_plan" {
```

Comment:
I don't think the code is setting up anything related with logdna / sysdig here.